package-lock.json

```diff
@@ -1910,14 +1910,20 @@
       "integrity": "sha512-QvO1J9drc7K2SeBrHuO5/8nQ1QTks/OOG+vwva4lM7T28pPgf556snMpPE/3KE6uD2UJky8I4A5dNRNbqAXiDw==",
       "dev": true,
       "requires": {
-        "axios": "0.21.1",
+        "axios": "0.19.0",
```

Comment:
you might wanna run npm install again, it seems to have dropped the force-resolutions :(

Reply:
eughhhhh. Good spot @andrewscfc

Reply:
hmm running npm install isnt changing anything, I think its working as expected as axios 0.21.1 is declared in the dependencies object below?

Reply:
hmm so weird, this occasionally seems to switch formats `npx npm-force-resolutions` updates it to an invalid format, then `npm install` updates it again to a valid format, but not always in the same way :( 

Anyway nothing to do with this PR.

Reply:
I just ran `npx npm-force-resolutions` and that seemed to worked. Have updated this PR